airflow/models/dag.py

```diff
@@ -2029,7 +2029,6 @@ def get_serialized_fields(cls):
                 'user_defined_filters',
                 'user_defined_macros',
                 'partial',
-                '_old_context_manager_dags',
```

Comment:
@kaxil can you take a look?

Reply:
Yeah it is safe, was just a duplicate from L2025